e2e/attach.test.ts

```diff
@@ -20,7 +19,7 @@ test('allow to attach to an existing session', async () => {
     await browser.url('http://guinea-pig.webdriver.io')
     expect(await browser.getTitle()).toBe('WebdriverJS Testpage')
 
-    const otherBrowser: Browser<'async'> = await attach(browser)
+    const otherBrowser = await attach(browser)
```

Comment:
just fyi: a lot of this stuff is fixed in #9256 which I hope to be able to finish this week

Reply:
I'll revert these changes in order to avoid conflicts